x/twap/helpers_test.go

```diff
@@ -0,0 +1,18 @@
+package twap
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/osmosis-labs/osmosis/v10/osmoutils"
+	"github.com/osmosis-labs/osmosis/v10/x/twap/types"
+)
+
+// GetAllHistoricalTimeIndexedTWAPs returns all historical TWAPs indexed by time.
+func (k Keeper) GetAllHistoricalTimeIndexedTWAPs(ctx sdk.Context) ([]types.TwapRecord, error) {
```

Comment:
N.B: these methods are only used in tests so I moved them into a file suffixed by `_test.go` to avoid having them exposed in the rest of the system.